src/main/scala/ai/privado/exporter/ExporterUtility.scala

```diff
@@ -45,21 +47,39 @@ object ExporterUtility {
     sourceId: String = "",
     taggerCache: TaggerCache = new TaggerCache()
   ): List[DataFlowSubCategoryPathExcerptModel] = {
+    val lang     = MetricHandler.metricsData("language")
+    val isPython = lang.toString().contains(Languages.PYTHONSRC)
+
     val sizeOfList = nodes.size
     nodes.zipWithIndex.flatMap { case (node, index) =>
       val currentNodeModel = convertIndividualPathElement(node, index, sizeOfList)
       if (
         index == 0 && node.tag.nameExact(Constants.catLevelOne).valueExact(CatLevelOne.DERIVED_SOURCES.name).nonEmpty
       ) {
-        val typeFullName = Traversal(node).isIdentifier.typeFullName.headOption.getOrElse("")
+        var typeFullName = Traversal(node).isIdentifier.typeFullName.headOption.getOrElse("")
+
+        // Temporary fix for python to match the typeFullName
+        if (isPython) {
+          typeFullName = typeFullName.stripSuffix(".<init>")
```

Comment:
Instead of handling it here, you can create a private function which takes typeFullName and return the formatted typeFullName based on language and use it here